extensions/amp-list/0.1/amp-list.js

```diff
@@ -251,13 +260,13 @@ export class AmpList extends AMP.BaseElement {
     // If a placeholder exists and it's taller than amp-list, attempt a resize.
     const placeholder = this.getPlaceholder();
     if (placeholder) {
-      this.attemptToFit_(placeholder);
+      this.attemptToFit_(placeholder).catch(() => {});
```

Comment:
Ah I see what you mean by catching every rejected Promise. You could also make attemptToFit resolve with a boolean instead of rejecting, or add a callback param.